diff --git a/bazel/repositories.bzl b/bazel/repositories.bzl
index 450d5cfe..5320a040 100644
--- a/bazel/repositories.bzl
+++ b/bazel/repositories.bzl
@@ -502,6 +502,12 @@ def _com_google_protobuf():
         name = "protoc",
         actual = "@com_google_protobuf_cc//:protoc",
     )
+    # Needed for ``bazel fetch`` to work with @com_google_protobuf
+    # https://github.com/google/protobuf/blob/v3.6.1/util/python/BUILD#L6-L9
+    native.bind(
+        name = "python_headers",
+        actual = "@com_google_protobuf//util/python:python_headers",
+    )
 
 def _com_github_grpc_grpc():
     _repository_impl("com_github_grpc_grpc")
